procServUtils/manage.py

```diff
@@ -118,8 +118,13 @@ def addproc(conf, args):
 
     _log.info("Writing: %s", cfile)
 
-    # ensure chdir is an absolute path
+    # ensure chdir and env_file are absolute paths
     args.chdir = os.path.abspath(os.path.join(os.getcwd(), args.chdir))
+    if args.env_file:
+        args.env_file = os.path.abspath(os.path.join(os.getcwd(), args.env_file))
```

Comment:
Would it be better to store `args.env_file` relative to `args.chdir` in our .conf file?  (where I assume it would normally be kept)  Certainly it would still need to be expanded to a absolute path for the `EnvironmentFile=`.

Reply:
I stored the absolute path just to be consistent with the `command` which is also stored as an absolute path. This way I can also check if the file exists, which could help avoid some errors.
While it's true that it will be usually located in the `args.chdir` folder, it could also be a system-wide file located for example in `/opt/` to serve multiple services, and in this case the relative path may not be ideal.
Furthermore, I expect the user to select the file writing its path relative to the directory where the `add` command is executed (`os.getcwd()`) to enable the use of autocompletion, so converting it to be relative to `chdir` may not be the most natural thing to do, in my opinion.